acceptance/bundle/exec/cmd-not-found/output.txt

```diff
@@ -0,0 +1,5 @@
+
+>>> [CLI] bundle exec -- doesnotexist arg1 arg2 --flag1 --flag2
```

Comment:
Is this different from 'bundle exec -- --help' test in /basic/?

Reply:
Fair enough, removed this test.